packages/api/src/controllers/stripe.js

```diff
@@ -86,8 +87,10 @@ app.post("/migrate-users", async (req, res) => {
   }
 
   const [users] = await db.user.find(
-    {},
-    { limit: 9999999999, useReplica: false }
+    [sql`users.data->>'stripeCustomerId' IS NULL`],
+    {
+      useReplica: false,
```

Comment:
I think a high `limit` might be necessary here, or we will default to `100` [here](https://github.com/livepeer/livepeer-com/blob/049c0fa1442db71201f90ea5eed49e627e69973c/packages/api/src/store/table.ts#L98)

Unless 100 is definitely ok, I think it wouldn't hurt to have something like 1000 or 10000 here (we have 7.5k users total in the db, as of now)

Otherwise, if you explicitly want **no** limit, I think you can set an explicit `limit: null` here which will work as per the current code (there's another check before using it [here](https://github.com/livepeer/livepeer-com/blob/049c0fa1442db71201f90ea5eed49e627e69973c/packages/api/src/store/table.ts#L135))

Reply:
ah yeah good call. I'll add back the limit.